docs/references/express/overview.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: Clerk Express.js SDK
+description: Learn how to integrate Express.js into your Clerk application.
+---
+
+# Clerk Express.js SDK
+
+## Set up Clerk Express.js
+
+<Steps>
+
+### Create a Clerk application
+
+You need to create a Clerk application in your Clerk Dashboard before you can set up Clerk Express.js. For more information, check out our [Set up your application](/docs/quickstarts/setup-clerk) guide.
+
+### Install `@clerk/express`
+
+To get started using Clerk with Express.js, add the [SDK](https://www.npmjs.com/package/@clerk/express) to your project:
+
+<CodeBlockTabs type="installer" options={["npm", "yarn", "pnpm"]}>
+```sh filename="terminal"
+npm install @clerk/express
+```
+
+```sh filename="terminal"
+yarn add @clerk/express
+```
+
+```sh filename="terminal"
+pnpm add @clerk/express
+```
+</CodeBlockTabs>
+
+### Set environment variables
+
+<SignedIn>
+
+Add the following keys to your `.env.local` file. These keys can always be retrieved from the [API Keys](https://dashboard.clerk.com/last-active?path=api-keys) page of your Clerk Dashboard.
+
+</SignedIn>
+
+<SignedOut>
+
+1. Navigate to the Clerk Dashboard.
+2. In the navigation sidebar, select [API Keys](https://dashboard.clerk.com/last-active?path=api-keys).
+3. In the **Quick Copy** section, copy your Clerk publishable and secret key.
+4. Paste your keys into your `.env.local` file.
+
+The final result should resemble the following:
+
+</SignedOut>
+
+  ```sh filename=".env.local"
+  NEXT_PUBLIC_CLERK_PUBLISHABLE_KEY={{pub_key}}
+  CLERK_SECRET_KEY={{secret}}
+  ```
+
+</Steps>
+
+## Available methods
+
+The Clerk Express SDK exports the following:
+
+- `@clerk/backend` - You can use [Clerk's JavaScript Backend SDK](/docs/references/backend/overview) to interact with the Clerk API.
+- `clerkMiddleware()` - The [`clerkMiddleware()`](/docs/references/nextjs/clerk-middleware) helper integrates Clerk authentication into your Next.js application through Middleware.
```

Comment:
It says here that this is specifically for nextjs, but there's an example above that uses it purely in an express app and makes no mention of nextjs. Is this specific to nextjs, even within the express SDK? If so, we need to call that out in the example above and provide an alternative for those not using nextjs.

Reply:
so is using `clerkMiddleware()` with express is specifically for only nextjs applications? @EmmanouelaPothitou

Reply:
it's incorrect. The `clerkMiddleware()` is un-related to NextJS apps. it's the core middleware required to use the `getAuth()`, ... core functionality of the `@clerk/express` in any application.